CRM/Event/Form/Participant.php

```diff
@@ -258,7 +258,7 @@ public function preProcess() {
       $this->_id = NULL;
     }
     else {
-      $this->_id = CRM_Utils_Request::retrieve('id', 'Positive');
+      $this->_id = CRM_Utils_Request::retrieve('id', 'Positive', $this);
```

Comment:
@eileenmcnaughton @pfigel are these changes strickly needed for this it feels like it goes back to the way of not allowing opening in new tabs or something

Reply:
@seamuslee001 AIUI the other changes in this PR don't address all the issues found in https://lab.civicrm.org/dev/report/issues/16 without reverting the participant bit in #14244.

Reply:
@seamuslee001 it's a partial revert of a change already in the rc (I didn;t revert the contribution page at this stage)

Reply:
Specifically we are reverting this https://github.com/civicrm/civicrm-core/pull/14244/files#diff-63338fa600c17c14a171ac3033cccdedL261